### PR TITLE
Faqpagefix

### PR DESCRIPTION
**Pull request by Nielsencu**
Used the material-ui accordion component instead of using vanilla react.

### CODE REVIEW

File: fintech-month/src/pages/faq/FaqPage.js [hunk 1]
```diff
@@ -1,7 +1,225 @@
...
+          <Typography className={classes.questions}>
+            Is it compulsory for all hackathon participants to join in the Live Stream?
+          </Typography>
+          <Typography className={classes.answers}>
+            Yes. Come join us for an exciting lineup of pitches made by our top 8 finalist teams to a panel of external judges and get to claim your e-vouchers from us at the end of the session!
+          </Typography>
+        </AccordionDetails>
+      </Accordion>
+
+      <Accordion disabled>
```
Comment (Wincenttjoi) [inline fintech-month/src/pages/faq/FaqPage.js:214]: U forgot to remove this section

Comment (Nielsencu) [reply fintech-month/src/pages/faq/FaqPage.js]: Removed the disabled accordion component!

File: fintech-month/src/pages/faq/FaqPage.js [hunk 2]
```diff
@@ -1,7 +1,217 @@
...
+    padding: '0 16px'
+  
+  },
+}));
+
+export default function FaqPage() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
```
Comment (Wincenttjoi) [inline fintech-month/src/pages/faq/FaqPage.js:39]: you may want to explore `<Container>` which makes the page more compressed to the center, will look nicer. And keep exploring the CSS to make the page nicer!

Comment (Wincenttjoi) [reply fintech-month/src/pages/faq/FaqPage.js:39]: Try to replace `<div>` to `<Container>` and see what happens!

Comment (Nielsencu) [reply fintech-month/src/pages/faq/FaqPage.js:39]: It becomes centered, very cool!
image

**Review by thewongwayround — APPROVED** [review]
looks good to me

**Review by mqchua — APPROVED** [review]

**Pull request merged**